kratos/python/add_mesh_to_python.cpp

```diff
@@ -750,11 +672,10 @@ void  AddMeshToPython(pybind11::module& m)
     .def("CalculateOnIntegrationPoints", CalculateOnIntegrationPointsVector<Condition>)
     .def("CalculateOnIntegrationPoints", CalculateOnIntegrationPointsMatrix<Condition>)
     // GetValuesOnIntegrationPoints
-    .def("GetValuesOnIntegrationPoints", GetValuesOnIntegrationPointsDouble<Condition>)
-    .def("GetValuesOnIntegrationPoints", GetValuesOnIntegrationPointsArray1d<Condition>)
-    .def("GetValuesOnIntegrationPoints", GetValuesOnIntegrationPointsVector<Condition>)
-    .def("GetValuesOnIntegrationPoints", GetValuesOnIntegrationPointsMatrix<Condition>)
+    .def("GetValuesOnIntegrationPoints", GetValuesOnIntegrationPoints<Condition>)
     // SetValuesOnIntegrationPoints
+    .def("SetValuesOnIntegrationPoints", SetValuesOnIntegrationPointsVector<Condition>)
```

Comment:
this seems unrelated, wasn't this in the other PR?

Reply:
There was a conflict and I did a bad merge. Thanks now it seems to be good.

Reply:
:+1: